BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift

```diff
@@ -248,19 +248,6 @@ public final class GiniBankConfiguration: NSObject {
      Sets the continue button text in the navigation bar on the review screen.
      */
     @objc public var navigationBarReviewTitleContinueButton = ""
```

Comment:
I think navigationBarReviewTitleContinueButton can be removed as well 😉

Reply:
I will check the button resources that we have on ScreenAPICoordinator, and if they are not used, I will remove all of them

Reply:
It seems that they are used in Xamarin code, so it should stay :D